spec/service_consumers/pact_helper.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require 'pact/provider/rspec'
+
+Pact.service_provider "VA.gov API" do
+  honours_pact_with 'VA.gov' do
```

Comment:
Might change from considering VA.gov as a single consumer to each app being individual consumers.